sdk/src/main/java/io/dapr/client/DaprHttp.java

```diff
@@ -153,26 +153,26 @@ public int getStatusCode() {
    * Invokes an API asynchronously without payload that returns a text payload.
    *
    * @param method        HTTP method.
-   * @param urlString     url as String.
+   * @param path          Array of path segments ("/a/b/c" maps to ["a", "b", "c"]).
    * @param urlParameters URL parameters
    * @param headers       HTTP headers.
    * @param context       OpenTelemetry's Context.
    * @return Asynchronous text
    */
   public Mono<Response> invokeApi(
       String method,
-      String urlString,
+      String[] path,
```

Comment:
suggestion - `pathSegments` might be a clearer name. Path is already a concept that's defined by URIs and most people wouldn't think of it as an array

Reply:
I can rename it.